From aa879239b6ea631bdad8ab48b52b09a262e17da4 Mon Sep 17 00:00:00 2001
From: Dave Eddy <dave@daveeddy.com>
Date: Thu, 21 Sep 2017 12:30:31 -0400
Subject: [PATCH] OS-6309 test-alias.js: add more verbose test messages
 Reviewed by: Patrick Mooney <patrick.mooney@joyent.com> Reviewed by: Josh
 Wilsdon <josh@wilsdon.ca> Approved by: Josh Wilsdon <josh@wilsdon.ca>

---
 src/vm/tests/test-alias.js | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/src/vm/tests/test-alias.js b/src/vm/tests/test-alias.js
index 88c6b2ee..46362028 100644
--- a/src/vm/tests/test-alias.js
+++ b/src/vm/tests/test-alias.js
@@ -1,4 +1,4 @@
-// Copyright 2015 Joyent, Inc.  All rights reserved.
+// Copyright 2017 Joyent, Inc.
 //
 // Tests for VM.lookup()
 //
@@ -81,7 +81,7 @@ function setAndCheckAlias(t, uuid, alias, base64, callback)
         }, function (cb) {
             // check that zonecfg value is correct
             getZonecfgAlias(uuid, function _checkZonecfgAlias(err, zalias) {
-                t.equal(zalias, base64);
+                t.equal(zalias, base64, 'alias the same in zonecfg');
                 if (err) {
                     t.ok(false, 'failed to get alias from zonecfg: '
                         + err.message);
@@ -98,7 +98,8 @@ function setAndCheckAlias(t, uuid, alias, base64, callback)
                     return;
                 }
 
-                t.equal((obj.alias == undefined) ? '' : obj.alias, alias);
+                t.equal((obj.alias == undefined) ? '' : obj.alias, alias,
+                    'alias the same from VM.load');
                 cb();
             });
         }
-- 
2.21.0

